templates/includes/sidebar.html

```diff
@@ -1,4 +1,16 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a></li>
+    <li>
+      <form action="{% url 'logout' %}" method="get">
+        <button type="submit">Logout</button>
+      </form>
```

Comment:
So, if you've decided to use form here - I have a concern on method, that you've decided to use.
Even though the logout button doesn't submit any data, it's still recommended to use the `POST` method here for the reasons mentioned below:
1. `GET` requests should be safe and idempotent, meaning they should not have any side effects when called multiple times. Logging out is not an idempotent action because it changes the state of the application.
2. I know you may not be familiar with some RESTful principles, but In a RESTful architecture, `POST` is used for actions that create or update a resource or cause a state change, while `GET` is used for read-only operations.
3. Using a `POST` request, you'll have to use `csrf-token`. A CSRF token for the logout action provides an additional layer of security against cross-site request forgery attacks.

Reply:
I actually wanted to refactor it, but part of code truncated(I mean that I just didn't copy entire refactored fragment and somehow passed part of it) that's why I deleted token, but left button. Still, thanks for clarifying, I will certainly use this info in future.
I refactored it the way I wanted, but wanna ask if I understood right, I should've changed the code like this?
```
      <form action="{% url 'logout' %}" method="post">
          {% csrf_token %}
        <button type="submit">Logout</button>
      </form>
```
and this would've been great for RESTful architecture(since it changes state right here) and provide some additional 
csrf-defence, right? But I have a question, then, if we use get for making these changes, like in any provided code, for example, in this if:
```
  {% if user.is_authenticated %}
    <li>User: <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a></li>
    <li>
      <a href="{% url 'logout' %}">Logout</a>
    </li>
  {% else %}
    <li>
      <a href="{% url 'login' %}">Login</a>
    </li>
  {% endif %}
```
we still change the state of user, by log-outing, so is this some miss, or it could be provided that way? And, if I understood correctly, my previous idea with logging out via [POST](https://github.com/mate-academy/py-taxi-service-authentication/pull/224/files/0f5c8bb21b9f8c238f2f85de9b6bf36562af869e#diff-08be20dc6785cf3568b16e27b7991a6e59eeac717effb5ccfda3de8280571b2d) was as RESTful as needed?